packages/venia-ui/lib/components/FilterModal/filterModal.js

```diff
@@ -58,12 +59,20 @@ const FilterModal = props => {
         [filterApi, filterItems, filterNames, filterState]
     );
 
+    const clearAll = filterState.size ? (
+        <div className={classes.action}>
+            <LinkButton type="button" onClick={handleReset}>
```

Comment:
This button needs a focus style/outline.

[![Image from Gyazo](https://i.gyazo.com/75796a523e8bca322323d996c91f5598.gif)](https://gyazo.com/75796a523e8bca322323d996c91f5598)